### PR TITLE
Improve whole Readme: wording/grammar/…

### PR DESCRIPTION
**Pull request by rugk**
* improved English grammar/typos/spelling errors
* fixed capitalization: abbreviations (API) and own project name (Croodle)
* improved the first wording a little, simplified, …
* gender-neutral referring to user via `them`
* tests can run under Chrome _and_ Chromium (actually you could likely make them work in Firefox, too)

### CODE REVIEW

**Comment by jelhan** [comment]
Thanks a lot!

> tests can run under Chrome and Chromium (actually you could likely make them work in Firefox, too)

You are right, but it's not configured like that. `ember test --server` will run tests against Chrome only. `ember test` will run tests against Chrome and Firefox. Chromium won't be used by default (except maybe if it's configured as an alias on OS level). This is configured in `testem.js`. Would be great if you could update docs accordingly.

**Comment by rugk** [comment]
Ehm, I have no ideas/detailed knowledge of that testing thing and what you've just said. It's likely easier for you to update the docs. :smile:

File: README.md [hunk 1]
```diff
@@ -79,17 +79,17 @@ to save your polls:
 
 ### Ember
 
-`ember test --server` is the prefered way to run tests in Chrome. Results are reported in command-line and browser. Files are watched for changes.
+`ember test --server` is the prefered way to run tests in Chrome/Chromium. Results are reported in command-line and browser. Files are watched for changes.
```
Comment (jelhan) [inline README.md]:
```suggestion
`ember test --server` is the prefered way to run tests in Chrome. Results are reported in command-line and browser. Files are watched for changes.
```

File: README.md [hunk 2]
```diff
@@ -79,17 +79,17 @@ to save your polls:
...
 
-If you like to run tests only once in a headless Chrome execute `ember test`. This is also used in CI.
+If you like to run tests only once in a headless Chrome/Chromium execute `ember test`. This is also used in CI.
```
Comment (jelhan) [inline README.md]:
```suggestion
If you like to run tests only once in a headless Chrome and Firefox execute `ember test`. This is also used in CI.
```

**Pull request merged**